server/routes/barcode/contacts/ChooseContactController.test.ts

```diff
@@ -34,24 +32,17 @@ const aContact = (): Contact => {
   }
 }
 
-const aPrisonAddress = (): PrisonAddress => {
-  return {
-    premise: 'HMP Leeds',
-  }
-}
-
 describe('ChooseContactController', () => {
   let chooseContactController: ChooseContactController
 
   beforeEach(() => {
-    chooseContactController = new ChooseContactController(
-      contactService as unknown as ContactService,
-      prisonRegisterService as unknown as PrisonRegisterService
-    )
+    chooseContactController = new ChooseContactController(recipientFormService as unknown as RecipientFormService)
```

Comment:
Minor (and I'm not proposing you change it), but I think you can do this once without a beforeEach. Do we need to re-instantiate the controller between each test? Might be cleaner like this?:
```
describe('ChooseContactController', () => {
  const chooseContactController = new ChooseContactController(recipientFormService as unknown as RecipientFormService)
```